### ISSUE TITLE
Another broadcast inconsistency with Base

### ISSUE THREAD

**Issue by bshall** [JuliaArrays/StaticArrays.jl#199]
In Base you get
```julia
broadcast(+, [1], []) == [] # true
broadcast(+, [], [1]) == [] # true
```
but in `StaticArrays` you get
```julia
broadcast(+, SVector(1), SVector()) === SVector() # true
broadcast(+, SVector(), SVector(1)) === SVector() # DimesionMismatch error
```

**Comment by wsshin** [comment]
@bshall, this has been fixed by https://github.com/JuliaArrays/StaticArrays.jl/pull/274.  Could you close this issue?

**Comment by c42f** [comment]
Sorry, I forgot to close this off.  Github should have been auto-closed this, but perhaps it needs something like `fixes #197, fixes #199 ...`?  Anyway, closing now.
